### PR TITLE
v5r1 fusion kernels for inference

### PR DESCRIPTION
**Pull request by zjing14**
- Add Ops, including Bias, Activation, Resize+Add, and MaxPool
- Extract the Write-out stage from GridwiseGemm, treat Write-Out and Gridwise as separate Ops
- Merge after #36

### CODE REVIEW

File: composable_kernel/include/utility/amd_buffer_addressing.hpp [hunk 1]
```diff
@@ -537,6 +537,7 @@ __device__ void amd_buffer_store_impl(const typename vector_type<T, N>::type src
         }
         else if constexpr(N == 8)
         {
+#if 0
```
Comment (asroy) [inline composable_kernel/include/utility/amd_buffer_addressing.hpp:540]: something wrong with ```llvm_amdgcn_raw_buffer_store_fp16x4```?

File: composable_kernel/include/utility/utility.hpp [hunk 2]
```diff
@@ -9,6 +9,8 @@ __device__ index_t get_thread_local_1d_id() { return threadIdx.x; }
 
 __device__ index_t get_block_1d_id() { return blockIdx.x; }
 
+//__device__ index_t get_block_1d_id() { return gridDim.x - 1 - blockIdx.x; }
```
Comment (asroy) [inline composable_kernel/include/utility/utility.hpp:12]: please remove

File: composable_kernel/include/utility/config.hpp [hunk 3]
```diff
@@ -78,7 +78,7 @@
 
 // experimental implementation
 #ifndef CK_EXPERIMENTAL_USE_BUFFER_LOAD_OOB_CHECK_OFFSET_TRICK
-#define CK_EXPERIMENTAL_USE_BUFFER_LOAD_OOB_CHECK_OFFSET_TRICK 0
+#define CK_EXPERIMENTAL_USE_BUFFER_LOAD_OOB_CHECK_OFFSET_TRICK 1
```
Comment (asroy) [inline composable_kernel/include/utility/config.hpp]: Why using this? for performance or correctness?

**Review by asroy — CHANGES_REQUESTED** [review]
```

CMakeFiles/device_gemm_instance.dir/__/device_operation/device_gemm_xdl_instance_f32_f32_f32_mk_nk_mn.cpp.o: In function `bfloat16_to_float':
/opt/rocm-4.1.0/hip/include/hip/hcc_detail/hip_vector_types.h:373: multiple definition of `bfloat16_to_float'
CMakeFiles/device_gemm_instance.dir/__/device_operation/device_gemm_xdl_instance_f32_f32_f32_mk_kn_mn.cpp.o:/opt/rocm-4.1.0/hip/include/hip/hcc_detail/hip_vector_types.h:373: first defined here
CMakeFiles/device_gemm_instance.dir/__/device_operation/device_gemm_xdl_instance_f32_f32_f32_mk_nk_mn.cpp.o: In function `size':
/root/workspace/composable_kernel/external/rocm/include/bfloat16_dev.hpp:68: multiple definition of `float_to_bfloat16'
```

Please fix the build issue for ```ckProfiler```

File: composable_kernel/include/utility/config.hpp (hunk 3 above)
Comment (zjing14) [reply composable_kernel/include/utility/config.hpp]: Yes. Used this for correctness issue.

**Pull request merged**